Constellation/Node/Storage/Sqlite.hs

```diff
@@ -73,8 +71,11 @@ load c k = do
         (Only k)
     return $ case rs of
         [Only b] -> Right $ decode $ BL.fromStrict b
-        []       -> Left "load: No payload found"
-        _        -> Left "load: More than one payload found"
+        -- []       -> Left "load: No payload found"
+        -- _        -> Left "load: More than one payload found"
+        -- TODO: In testStorage, don't rely on the presence of the strings below
+        []       -> Left "Key not found in SQLite"
```

Comment:
@patrickmn actually I was thinking of changing the error text for all of the storage types to "payload not found" as you originally had here, since users seem to be confusing "key" with "private/public key".